<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vijayp" ignored="0" name="Test A2" tests="3" failures="1" timestamp="02 Nov 2017 06:28:34 GMT" time="0.023" errors="0">
  <testcase name="testA2_1" time="0.002" classname="testSuiteA.TestA2"/>
  <testcase name="testA2_2" time="0.002" classname="testSuiteA.TestA2">
    <failure type="java.lang.AssertionError" message="failed test case ">
      <![CDATA[java.lang.AssertionError: failed test case 
	at testSuiteA.TestA2.testA2_2(TestA2.java:16)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testA2_2 -->
  <testcase name="testA2_3" time="0.003" classname="testSuiteA.TestA2"/>
</testsuite> <!-- Test A2 -->
